src/widgetastic/widget.py

```diff
@@ -842,8 +842,11 @@ def fill(self, value):
 class ClickableMixin(object):
 
     @logged()
-    def click(self):
-        return self.browser.click(self)
+    def click(self, handle_alert=True):
```

Comment:
We probably do not want to have the alerts checked on every click because it takes 1-2s to do it. I would recommend using the approach that None means do nothing and True/False will trigger the alert handling and keeping None as default.

I have sped up our navigation by removing alert checks where it was not necessary, now we would get back to that slow speed :)